### PR TITLE
handle floats / money

### PR DESCRIPTION
**Pull request by rafipiccolo**
console.log(MoneyToLetter(1, 'euro'));
// un euro

console.log(MoneyToLetter(12.05, 'euro'));
// douze euros et cinquante centimes

console.log(MoneyToLetter(12.5, 'euro'));
// douze euros et cinquante centimes

console.log(MoneyToLetter(12.567, 'euro'));
// douze euros et cinq cent soixante-sept centimes

### CODE REVIEW

**Comment by rafipiccolo** [comment]
oups. should have been
if (virgulepart.length == 1)
instead of
if (virgulepart > 0 && virgulepart < 10)

File: index.js [hunk 1]
```diff
@@ -176,6 +176,29 @@ function NumberToLetter( nombre ){
...
+            virgulepart = `${virgulepart * 10}`;
+        // remove sub cents : 0,5123 => 0,51
+        virgulepart = Math.round(virgulepart * 100) / 100;
+        virgulepartstring = NumberToLetter(virgulepart);
+    }
+
+    return `${integerpartstring} ${devise}${integerpart <= 1 ? '' : 's'}${virgulepartstring ? ` et ${virgulepartstring} centimes` : ''}`;
+}
+
+exoirts.MoneyToLetter = MoneyToLetter;
```
Comment (Spixz) [inline index.js:201]: There is a synthax error with the export.

**Comment by cmammar** [comment]
Thanks for the add of this functionnality.
If someone want to add this branch to his project you can do:
`npm install yagoubigithub/convertir-nombre-lettre#pull/2`

**Comment by Spixz** [comment]
Thanks for the add of this functionnality.
If someone want to add this branch to his project you can do:
`npm install yagoubigithub/convertir-nombre-lettre#pull/2`

**Pull request merged**